app/controllers/organizations_controller.rb

```diff
@@ -191,4 +192,12 @@ def get_pr_relations
 
     PullRequestRelation.by_organizations(@organization.id).within_month_limit(@month_limit)
   end
+
+  def get_team_id
+    if @team && permitted_params[:froggo_team] == "true"
+      return @team.id
+    end
+
+    nil
+  end
```

Comment:
Esto lo podrías hacer más simple así:

```suggestion
  def get_team_id
    permitted_params[:froggo_team] ? @team&.id : nil
  end
```
(no estoy seguro si es necesario ese `== true`, pero me imagino que no. El `&.` va a retornar `nil` si no existe el `@team`

Reply:
no cachaba eso del & ! gracias por el tip